internal/usecase/picker/model.go

```
@@ -312,60 +312,64 @@ func rowsForSession(session SessionView, config SelectionConfig) []row {
 	}
 	if config.Mode == SelectMovePaneTab && len(rows) == 1 && !session.CannotAcceptMoves {
 		return nil
 	}
 	return rows
 }
 
+func remoteStoppedOrdinalSelector(index int, rawName string, tabCount int) (domain.TabSelector, bool) {
+	if tabCount > math.MaxUint16 {
+		return domain.TabSelector{}, false
+	}
+	return domain.NewOrdinalTabSelector(uint16(index), rawName, uint16(tabCount)), true
```

golangci-lint (2.12.2)

[high] G115: integer overflow conversion int -> uint16

(gosec)